src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -860,6 +860,20 @@ void GeneralPage::initLayout(GeneralPageView &layout)
         s.attachExtensionToAnyProcess, false,
         "Attempt to force the Chatterino Browser Extension to work in certain "
         "browsers that do not work automatically.\ne.g. Librewolf");
+    layout.addSubtitle("Additional extension IDs");
+    layout
+        .addDescription("A semicolon separated list of Chrome or Firefox "
+                        "extension IDs that are allowed to interact with "
+                        "Chatterino's browser integration (requires restart):"
+        ->setToolTip("Chatterino does not provide strong API guarantees.\nNote "
+                     "that specifying multiple extension IDs for different"
+                     "browsers might cause issues."
+    {
+        auto *edit =
+            this->createLineEdit(getSettings()->additionalExtensionIDs);
```

Comment:
Make use of the already-existing `Settings` variable
```suggestion
        auto *edit = this->createLineEdit(s.additionalExtensionIDs);
```